highlight/highlight
# Add headers to the logs table

## Summary
Update the logs table to use our common table component, and provide the table with headers to match the style of the traces table.

Note: this still uses the virtual table - we may want to consider building a wrapper around our table component to handle these types of table when converting the traces table.

## How did you test this change?
1) Load the logs table
- [ ] Confirm more logs can be fetched via the top row notification
- [ ] Confirm more loads are loaded when scrolling to the bottom
- [ ] Confirm logs row can be opened and closed 
- [ ] Confirm data in logs can be be copied or used to filter appropriately
2) Copy a link to a log, and navigate to the url
- [ ] Confirm the logs is scrolled down to
- [ ] Confirm the log is expanded by default

https://www.loom.com/share/26413f3719734be4be98cea85ab478b4?sid=60d88918-33fe-4878-a2e1-8c9aa8b0ea5e

## Are there any deployment considerations?
N/A

## Does this work require review from our design team?
Yes
